tests/RemoteLRSTest.php

```diff
@@ -374,6 +374,44 @@ public function testMoreStatementsWithAttachments() {
         $this->assertInstanceOf('TinCan\StatementsResult', $response->content, 'content');
     }
 
+    public function testRetrieveStatementWithFileUrlAttachments() {
+        $lrs = new RemoteLRS(self::$endpoint, self::$version, self::$username, self::$password);
+        $attachments = new Attachment(); 
+        $pdfUrl = 'http://www.adlnet.gov/wp-content/uploads/2013/10/xAPI_v1.0.1-2013-10-01.pdf';
```

Comment:
I realize this won't get fetched, but we probably don't want to point to an external resource, nor include another file in the repo for this purpose. See the `tests/files/image.jpg` which can be linked at https://github.com/RusticiSoftware/TinCanPHP/raw/master/tests/files/image.jpg (could also point at one of the raw github services).